nemo/collections/asr/parts/utils/diarization_utils.py

```diff
@@ -538,7 +538,11 @@ def make_json_output(self, uniq_id, diar_hyp, word_dict_seq_list, total_riva_dic
             total_riva_dict[uniq_id] = riva_dict
             prev_speaker = speaker
 
+        # note that we need to add the very last sentence.
```

Comment:
Just to inform other reviewers, this change is made because in the recent PR(https://github.com/NVIDIA/NeMo/pull/3897), [demsarjure](https://github.com/demsarjure) missed to put the very last line for transcription generation so I add it here.